docs/installation/installation-using-docker-linux.md

```diff
@@ -9,7 +9,7 @@ Take a look at the article about [Monorepo](../introduction/monorepo.md) for mor
 
 ## Requirements
 * [GIT](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-* [PHP](http://php.net/manual/en/install.unix.php)
+* [PHP 7.1 - 7.3](http://php.net/manual/en/install.unix.php)
```

Comment:
Hi, I would not mention the upper limit, we would need to change it every time we add support of new PHP :slightly_smiling_face:

Reply:
Hello @vitek-rostislav,
I got inspiration from native-installation.md where it is wrote same way :-) do you insist on updating it?

Reply:
yes please :innocent: I would rather get inspired by the line below in the same doc where we mention minimum version for Docker, so I would put it this way:

```
* [PHP](http://php.net/manual/en/install.unix.php)
	* At least version **7.1 or higher**
```

Reply:
Hello @vitek-rostislav,

I have updated it as requested and also rebased on actual master.